addons/mail/static/src/core/common/message_model.js

```diff
@@ -116,6 +116,9 @@ export class Message extends Record {
     scheduledDatetime = Record.attr(undefined, { type: "datetime" });
     onlyEmojis = Record.attr(false, {
         compute() {
+            if (this.isEmpty) {
+                return false;
+            }
```

Comment:
Isn't the compute "working" even without this check?
It should trivially return false.

Reply:
Test `"message considered deleted"` shows inconsistent size because `onlyEmojis` is true for some empty messages.

The compute method can be true when the message is empty, `bodyWithoutTags` can be whitespaces thus it things there are only emojis.

An alternative could be to see whether there's at least 1 emoji in `bodyWithoutTags`.